### DISCUSSION TITLE
Allow the user to select their own start time and date

### DISCUSSION THREAD

**Discussion by jamespaulsports** [add2cal/add-to-calendar-button#458] [Ideas / Requests]
Hi, I’ve been implementing the add to calendar button in my Wordpress site. On my site it allows a user to add a workout or cycling training plan to their calendar. This works okay but I would like to allow users to choose what date they add an individual workout or plan. Currently I have to determine what date and start/finish time the user can add it to their calendar on. That means they then have to go into apple/google calendars and manually amend. Could this be a future feature?

**Comment by jekuer** [comment]
Hi @jamespaulsports ,

Unfortunately, as of now, this is no feature we are aiming for, since it is a very niche case and theoretically can be handled even better (and more stable) on your side (like getting the user's individual setting and then adjusting the shortcode dynamically).
We will come up with a managed service soon, which also comes with an API that can be used for your case - however, this will also require some deeper implementation into your code. Let's see what the future brings - maybe the WordPress plugin gets way bigger. But this won't happen very soon, to be fair.

All the best,
Jens
